### DISCUSSION TITLE
Auto-include webpack vendors bundles

### DISCUSSION THREAD

**Discussion by Mattia-Re** [webpack/webpack#19045] [Q&A]
At this point I don't know where else to ask about this, but I've been stuck on this for months now which is driving me away from using webpack.

I'm using webpack's CodeSplittingPlugin which produces vendors bundles, is there a way to prevent the need to manually add and update vendor bundles script tags in HTML? Something like a bundle that has a static name and that I can add to all of my HTML documents which will auto load the vendor bundle for scripts that require it?

I don't want to use the HTMLWebpackPlugin because I'm using Razor views and not plain HTML.

**Comment by alexander-akait** [comment]
@iKingNinja You can get them using `stats` after compilations (like here - https://github.com/jantimon/html-webpack-plugin/blob/main/index.js#L1293), or create your own small plugin and automate it using the same logic, it depends on how you use webpack and how your application is built
